docs/root/version_history/current.rst

```diff
@@ -48,6 +48,7 @@ Bug Fixes
 * cluster: fixed the :ref:`cluster stats <config_cluster_manager_cluster_stats_request_response_sizes>` histograms by moving the accounting into the router
   filter. This means that we now properly compute the number of bytes sent as well as handling retries which were previously ignored.
 * hot_restart: fix double counting of `server.seconds_until_first_ocsp_response_expiring` and `server.days_until_first_cert_expiring` during hot-restart. This stat was only incorrect until the parent process terminated.
+* http: fix erroneous handling of invalid nghttp2 frames with a `NGHTTP2_ERR_REFUSED_STREAM` error.
```

Comment:
I would suggest saying something about the old and new behaviors.  I think that on REFUSED_STREAM envoy used to close the H2 upstream connection.  After this change, Envoy will terminate just the refused stream.